src/server/api/geo/getForestAgreementLayer.ts

```diff
@@ -6,33 +6,54 @@ import { ForestSource } from '@meta/geo'
 import { GeoController } from '@server/controller/geo'
 import Requests from '@server/utils/requests'
 
-export const getForestAgreementLayer = async (req: Request, res: Response) => {
+export const getForestAgreementLayer = async (
+  req: Request<
```

Comment:
Not sure if it's valid for your case but you can use also CycleRequest?

Reply:
Found under `meta/`

Reply:
I think for now we are not linking to a specific cycle, (only using ISO to clip country) but is something to be discussed.